README.md

```diff
@@ -244,82 +253,20 @@ On the first line, you are seeing a snapshot of the HTTP request that was receiv
- to include this action:
-
-**app/controllers/welcome_controller.rb**
-
-```ruby
-class WelcomeController < ApplicationController
-  def index
-
-  end
-end
-```
-
-Refresh the page again. New error! You should now see the following text (with some additional info): `WelcomeController#index is missing a template for request formats: text/html`. Rails is telling us that now it was able to find the controller and action we wanted, but once there, it didn't know what information to send back to the browser in the response body. It was expecting to find some HTML that it could send back for the browser to render, but it found nothing. Let's go create that HTML in our views directory.
-
-In your `app/views` directory, add a sub-directory called `welcome` and, within that directory, a file called `index.html.erb`. When creating views, we name the files the same as our action: `def index` to `index.html.erb`. And, those files will live in a directory with the same name as our controller: `WelcomeController` to `views/welcome/`. The path for your new file should be `app/views/welcome/index.html.erb`.
-
-In that file, add the following HTML:
-
-**app/views/welcome/index.html.erb**
-
-```html
-<h1>Welcome to the Task Manager</h1>
-
-<ul>
-  <li><a href="/tasks">Task Index</a></li>
-  <li><a href="/tasks/new">New Task</a></li>
-</ul>
-```
+## Returning All Tasks
 
-We have an h1 tag for our welcome message, then an unordered list (ul) with two list items (li) inside. If you are unfamiliar with HTML tags, try one of the HTML tutorials before continuing.
+### Adding the Route
 
-Inside of each li tag, we have an `<a>` tag. The href of the tag is the path where the link will go. In the first a tag, the path will be [http://localhost:3000/tasks](http://localhost:3000/tasks). The second path will be [http://localhost:3000/tasks/new](http://localhost:3000/tasks/new).
+Let's update our app to return something other than the default Rails welcome page - remember our goal is to send back JSON data that a frontend application could use.
 
-Refresh your browser and you should now see an error-free view! What do you think will happen if you click on one of these links? More errors - but that's ok! We can use these errors to help guide us in our development.
+Here's our goal. When our browser makes a GET request to `localhost:3000/tasks`, we want our API to return all of the tasks currently in the database.
 
-## Adding a Task Index
+We want to see something like this:
+<!-- Insert screenshot here -->
 
-From the home page of your app (localhost:3000/), click on the link for `Task Index`. You should see an error telling you that `No route matches [GET] "/tasks"`. Our app received a request that it was not set up to handle, so let's change that!
+Try navigating to `localhost:3000/tasks` in your own browser. You should see an error telling you that `No route matches [GET] "/tasks"`. Our app received a request that it was not set up to handle, so let's change that!
```

Comment:
I'm wondering -- should we be having them namespace as `api/v1` so as to start building the habit around that particular convention?

Reply:
Juliette and I discussed this, but we decided against it to keep things more straightforward for this first introduction. The current plan is to introduce namespacing during the advanced routing lesson, but that doesn't come until week 5. I'll start a discussion in slack to see if we're ok with the projects at the start not having namespaces.